docs/lang/articles/contribution/write_test.md

```diff
@@ -75,6 +75,8 @@ def test_log10():
     assert r[None] == 2
 ```
 
+In this case, Taichi will execute this test case on both `ti.cpu` and `ti.cuda` backends.
+
 And you may test against **all backends** by simply not specifying the
```

Comment:
nit: ...all (available?) backends... or backends (depending on the system)... ;/